.yarnrc.yml

```diff
@@ -0,0 +1,9 @@
+enableGlobalCache: true
+
+nodeLinker: node-modules
```

Comment:
A. couple of options worth adding:
1. `compressionLevel: 0` - instructs Yarn to NOT compress zip archives in the cache.Zip compression is somewhat useful for pnp to reduce git repository size, but even for pnp there are cases when uncompressed archives are better. And for node_modules they are almost always the preferred option. Without this option Yarn 3 install time takes 41secs for me, whilst Yarn 1 takes 31 secs. With this option Yarn 3 install time reduces to 26 secs for me in total.
2. `nmMode: hardlinks-local` - this is a new option in Yarn 3 which virtually does not have downsides. It enables usage of hardlinks inside node_modules for duplicate package copies, it should not break compatibility with any npm packages because they don't see any difference between hardlink and real file and it reduces total disk space used by the installed project for 1.7G to 1.4G approximately, and very slightly increases perfs, install time takes 24.5 secs for me. There is also `nmMode: hardlinks-global` mode where hardlinks point to the global store, but it has downsides - the files in the global store shared by all the projects and modifications inside node_modules of one project will be reflected in the other project that uses this mode.

Reply:
similar performance improvements for me with these options!